nimble_builder/shelf_builder.py

```diff
@@ -283,13 +417,14 @@ def cut_opening(self,
         sketch.add_rect(size_x, dim_y.tuple, center="X")
         self._shelf.cut_extrude(face, sketch, -depth)
 
-    def add_mounting_hole_to_bottom(self,
-                                    x_pos: float,
-                                    y_pos: float,
-                                    base_thickness: float,
-                                    hole_type: Literal["M3cs", "M3-tightfit", "base-only"],
-                                    base_diameter: float = 15
-                                    ) -> None:
+    def add_mounting_hole_to_bottom(
+        self,
+        x_pos: float,
+        y_pos: float,
+        base_thickness: float,
+        hole_type: Literal["M3cs", "M3-tightfit", "base-only"],
```

Comment:
Not a clue actually. This is only marked as new code because I changed its indenting. I will fix this at a later date as "tightfit" also seems to mean a hole that will be tapped by the screw.
Added task to #58